### PR TITLE
Fix service_logs API exposes unimplemented POST method for /api/servi…

### PR DESCRIPTION
**Pull request by Lir10**

### CODE REVIEW

**Comment by Lir10** [comment]
@jhernand ptal

File: model/service_logs/v1/cluster_logs_uuid_resource.model [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+		// Retrieved list of Cluster logs.
+		out Items []LogEntry
+	}
+
+    // Reference to the service that manages a specific Log entry.
+	locator LogEntry {
+		target LogEntry
+		variable ID
+	}
```
Comment (jhernand) [inline model/service_logs/v1/cluster_logs_uuid_resource.model:68]: I this really supported? I think there are no sub-resources of this resource.

Comment (Lir10) [reply model/service_logs/v1/cluster_logs_uuid_resource.model:68]: LogEntry is a class in `cluster_log_type.model`

Comment (jhernand) [reply model/service_logs/v1/cluster_logs_uuid_resource.model:68]: This locator translates into a `/api/service_logs/v1/clusters/{cluster_id}/cluster_logs/{entry_id}` endpoint. I think we don't have such an endpoint in the server, do we? If we don't have that endpoint then this locator can and should be removed.

Comment (Lir10) [reply model/service_logs/v1/cluster_logs_uuid_resource.model:68]: I see .. ok I'll remove that one. thanks

**Pull request merged**